sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFileFormat.scala

```diff
@@ -272,6 +273,14 @@ class ParquetFileFormat
       schema.forall(_.dataType.isInstanceOf[AtomicType])
   }
 
+  override def vectorTypes(
+      sparkSession: Option[SparkSession],
```

Comment:
do we really need the session parameter?

Reply:
I see, let's remove this for now.